Userland/Libraries/LibWeb/CSS/CSSPropertyRule.cpp

```diff
@@ -13,12 +13,12 @@ namespace Web::CSS {
 
 JS_DEFINE_ALLOCATOR(CSSPropertyRule);
 
-JS::NonnullGCPtr<CSSPropertyRule> CSSPropertyRule::create(JS::Realm& realm, FlyString name, FlyString syntax, bool inherits, Optional<FlyString> initial_value)
+JS::NonnullGCPtr<CSSPropertyRule> CSSPropertyRule::create(JS::Realm& realm, FlyString name, FlyString syntax, bool inherits, Optional<String> initial_value)
```

Comment:
Please move these changes into the first commit (and the ones in `CSSPropertyRule.h`)